scripts/ci

```diff
@@ -12,14 +12,5 @@ cd $(dirname $0)/..
 echo "Bootstrapping..."
 yarn --pure-lockfile install
 
-echo "Linting..."
```

Comment:
We should keep these in (this script is used by other steps separate to pull requests). Instead the `pull_request` trigger should be removed from the 'test` job in `./.drone.yaml` (at some point we should migrate the builds over to actions as well, but for the moment lets leave them with a basic check).